packages/react/src/components/Animation/Animation.tsx

```diff
@@ -24,42 +24,42 @@ export interface AnimationProps
   /** The name for the animation that should be applied, defined in the theme. */
   name?: string
 
-  /** The delay property specifies a delay for the start of an animation. Negative values are
```

Comment:
Most components don't reiterate the name of the property in the doc string, which I think is a good thing. I've removed it from Animation to make it consistent with the rest of the components.